code/modules/paperwork/pen.dm

```diff
@@ -104,21 +104,6 @@
 	icon_state = "fountainpen"
 	pen_color_shift = 0
 
-/obj/item/pen/attack(mob/living/M, mob/user)
```

Comment:
why is this removed?

Reply:
This is arguably out of scope, but when I was originally making this PR i had the renaming proc be a method on the pen. And figured that the `/attack` logic is horrid, and only exists to facilitate sleepy pen injections. There's no good reason for why it is not under `pen/sleepy/attack()`.
Sleepy pen still works; attacking people with regular pen no longer gives "you stab a guy with a pen" messages that only you (but not your victim!) and admins(!) can see.

Reply:
Sounds logical. Can you add this to the PR description and CL? Good change btw

Reply:
Done.